dtls/wrapper.py

```diff
@@ -0,0 +1,370 @@
+# -*- encoding: utf-8 -*-
+

```

Comment:
Well yes i meant coding:

https://www.python.org/dev/peps/pep-0263/   which also validates encoding ....

> ^[ \t\v]*#.*?coding[:=][ \t]*([-_.a-zA-Z0-9]+)

But I'll correct that. And also fix the line endings and exclude the wrapper from the \_\_init__.py

Regards